libdnf5-cli/progressbar/multi_progress_bar.cpp

```diff
@@ -105,10 +105,10 @@ std::ostream & operator<<(std::ostream & stream, MultiProgressBar & mbar) {
     text_buffer.clear();
 
     if (is_interactive && mbar.num_of_lines_to_clear > 0) {
-        text_buffer << tty::clear_line;
-        for (std::size_t i = 1; i < mbar.num_of_lines_to_clear; i++) {
-            text_buffer << tty::cursor_up << tty::clear_line;
+        if (mbar.num_of_lines_to_clear > 1) {
+            text_buffer << tty::cursor_up_multiple_lines(mbar.num_of_lines_to_clear - 1);
```

Comment:
I like the approach, but please don't do it like this via a new public class.

Perhaps just inline it with a comment what the sequences mean.

Reply:
It is much better and simpler, thank you.